src/zope/interface/tests/test_odd_declarations.py

```diff
@@ -214,7 +214,7 @@ class C2(C1):
 
 def test_suite():
     import doctest
-    suite = unittest.TestSuite()
-    suite.addTest(unittest.makeSuite(Test))
-    suite.addTest(doctest.DocTestSuite(odd))
-    return suite
+    return unittest.TestSuite((
+        unittest.defaultTestLoader.loadTestsFromName(__name__),
+        doctest.DocTestSuite(odd),
+    ))
```

Comment:
This one does have (and run) a doctest from a docstring. It *could* be ported (it's moderately lengthy) or we could leave it.